tests/test_pool.py

```diff
@@ -0,0 +1,24 @@
+import ssl
+
+import pytest
+
+from aapns.pool import Pool
+
+pytestmark = pytest.mark.asyncio
+
+
+async def test_bad_pool_size():
+    with pytest.raises(ValueError):
+        await Pool.create("https://localhost:1234", 0)
+
+
+async def test_bad_origin():
+    with pytest.raises(ValueError):
+        await Pool.create("just.host.name.not.ok")
```

Comment:
FIY there's [`.invalid`](https://en.wikipedia.org/wiki/.invalid) which seems to be exactly for this.

Reply:
Yay, cool! :)